### PR TITLE
Ev/implement data access interface#100

### PR DESCRIPTION
**Pull request by valetee**
Some of the stuff is incomplete, e.g. The copy constructors are not completely tested so they look really whack right now.

### CODE REVIEW

**Comment by shinniem** [comment]
Once the label test is added then good to merge

File: app/build.gradle [hunk 1]
```diff
@@ -34,7 +34,6 @@ dependencies {
     implementation 'com.google.android.material:material:1.1.0'
     implementation 'junit:junit:4.12'
     implementation 'junit:junit:4.12'
-    implementation files('libs/hsqldb.jar')
```
Comment (Arvind-Maan) [inline app/build.gradle:37]: is this supposed to be happening??

File: app/src/main/java/comp3350/mealbuddy/business/AccessEdible.java [hunk 2]
```diff
@@ -32,23 +30,27 @@ public void addEdible(Edible e) {
...
+    public void updateEdible(String edibleToUpdate, Edible e){
+        DAS.updateEdible(edibleToUpdate, e);
     }
 
-
     /*
      * removeEdible
      * Remove an edible from the database.
      * Parameters:
      *     @param e - The edible to remove.
```
Comment (Arvind-Maan) [inline app/src/main/java/comp3350/mealbuddy/business/AccessEdible.java:49]:
```suggestion
     *     @param e - The name of the edible to remove.
```

File: app/build.gradle [hunk 3]
```diff
@@ -34,7 +34,6 @@ dependencies {
     implementation 'com.google.android.material:material:1.1.0'
     implementation 'junit:junit:4.12'
     implementation 'junit:junit:4.12'
```
Comment (Arvind-Maan) [inline app/build.gradle:36]:
```suggestion
    implementation 'junit:junit:4.12'
    implementation files('libs/hsqldb.jar')
```

**Pull request merged**